api_tests/schemas/views/test_registration_schemas_list.py

```diff
@@ -14,13 +15,16 @@ class TestSchemaList:
     def test_schemas_list_crud(self, app):
 
         user = AuthUserFactory()
-        url = '/{}schemas/registrations/'.format(API_BASE)
-
+        url = '/{}schemas/registrations/?version=2.11'.format(API_BASE)
+        schemas = RegistrationSchema.objects.filter(schema_version=LATEST_SCHEMA_VERSION)
         # test_pass_authenticated_user_can_view_schemas
         res = app.get(url, auth=user.auth)
         assert res.status_code == 200
-        assert (len(res.json['data']) == RegistrationSchema.objects.filter(
-            active=True, schema_version=LATEST_SCHEMA_VERSION).count())
+
+        if waffle.switch_is_active('filter_schemas_registration_on_active'):
+            assert res.json['meta']['total'] == schemas.count()
```

Comment:
Since this PR doesn't have a migration adding the switch, this part of your test isn't being run since `waffle.switch_is_active('filter_schemas_registration_on_active')` is `False`. 

You can override waffle switches in tests to fake their behavior. Here's an example of where we do that: https://github.com/caseyrollins/osf.io/blob/develop/osf_tests/test_queued_mail.py#L191

Reply:
Note that even if this PR did have a migration adding the switch, the other part of the if statement wouldn't be run.